modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -278,29 +219,25 @@ def _get_cpus_per_actor(num_actors):
 
 def _get_num_actors(num_actors):
     """
-    Get number on actors to create.
+    Get number of actors to create.
 
-    In case `num_actors` is 'default_train' string,
-    integer number of actors will be computed  by condition
-    2 CPUs per 1 actor, in case 'default_predict' 8 CPUs per 1 actor.
+    In case `num_actors` is None, integer number of actors
+    will be computed by condition 2 CPUs per 1 actor.
 
     Parameters
     ----------
-    num_actors : int or {'default_train', 'default_predict'}
-        Number of actors to create or str to set defaults.
+    num_actors : int or None
```

Comment:
```suggestion
    num_actors : int, default: None
```

Reply:
Used
```python
num_actors : int, optional
```
with expanding of `num_actors` parameter description